src/com/itacademy/eshop/product/Clothing.java

```diff
@@ -0,0 +1,9 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.product.types.Category;
+
+public abstract class Clothing extends Product {
```

Comment:
There is no need of this abstract class if it does not have any abstract fields or methods. You can just remove this Clothing class and extend shirt with product directly